### PR TITLE
Makefile etc. tidy ups - can run on OS X now

### PR DESCRIPTION
**Pull request by timlinux**
@lucernae please review & test local before I merge

### CODE REVIEW

File: deployment/docker-compose.yml [hunk 1]
```diff
@@ -70,38 +77,4 @@ web: &web
...
-    - devdb:db
-    - devdbbackup:dbbackup
-
-devweb:
-  <<: *web
-  links:
-    - devuwsgi:uwsgi
-  ports:
-    - "61102:8080"
+    - "61100:8080",
```
Comment (lucernae) [inline deployment/docker-compose.yml]: Rogue comma here...

**Comment by lucernae** [comment]
I still need the dev env for now.
It is just too painful too debug from inside docker container.
I don't like remote debug either because I need to add pydevd and debug line each time I have to debug something.
I will probably adapt this into another PR.
Also, once docker-compose 1.5 is out, we can extends docker-compose.yml configuration file and override it using development yml conf file. It would save lives :p.

**Comment by ismailsunni** [comment]
@timlinux @lucernae Does the change affect the running in Ubuntu?

**Comment by lucernae** [comment]
@ismailsunni you can skip the docker-machine part when you're developing in Linux, because the you are already in the machine

**Comment by lucernae** [comment]
Closing this pull request, because the changes is applied on another PR.

**Pull request closed without merging**